docs/pages/blog/mui-core-v5.md

```diff
@@ -0,0 +1,546 @@
+```
+
+We have added support for [responsive values](/components/grid/#responsive-values) on all the props:
+
+```jsx
+<Grid container spacing={{ xs: 2, md: 3 }} />
+```
+
+We have added support for a different [number of columns](/components/grid/#columns) than 12:
+
+```jsx
+<Grid container columns={16}>
+```
+
+We have added an alternative implementation that uses [CSS grid](/components/grid/#css-grid-layout):
+
+```jsx
+<Box display="grid" gridTemplateColumns="repeat(12, 1fr)" gap={2}>
+  <Box gridColumn="span 8">
+    <Item>xs=8</Item>
+  </Box>
+  <Box gridColumn="span 4">
+    <Item>xs=4</Item>
+  </Box>
+  <Box gridColumn="span 4">
+    <Item>xs=4</Item>
+  </Box>
+  <Box gridColumn="span 8">
+    <Item>xs=8</Item>
+  </Box>
+</Box>
+```
+
+### More Material Design icons
+
+The Material Design team at Google has released 600 new icons in five different themes since we released v4.
+We have made them [available](/components/material-icons/) in the `@mui/icons-material` package.
+
+### Stack
+
+We have introduced a new `<Stack>` component.
+It handles one-dimensional layouts, it's similar to how Figma handles auto-layout.
+
+<a href="/components/stack/"><img src="/static/blog/mui-core-v5/stack.png" alt="" style="width: 532px; margin-bottom: 16px;" /></a>
+
+> Note that you might already be using `<Box display="flex" gap={1}>` to solve the same problem.
+> However [browser support](https://caniuse.com/flexbox-gap) for the flexbox `gap` CSS property is lacking in Safari.
+
+You can find [more details](/components/stack/) in the documentation.
+
+### Promotion from the lab
+
+We have promoted six component from the lab, after over two years iterating on feedback:
+
+- [Autocomplete](/components/autocomplete/)
+- [Pagination](/components/pagination/)
+- [Rating](/components/rating/)
+- [Skeleton](/components/skeleton/)
+- [Speed Dial](/components/speed-dial/)
+- [Toggle Buttons](/components/toggle-button/)
+
+### New in the lab
+
+The lab hosts the incubator components that are not yet ready to move to the core.
+The main difference between the lab and the core is how the components are versioned.
+Having a separate lab package allows us to release breaking changes when necessary while the core package follows a [slower-moving policy](/versions/#release-frequency).
+
+The following components are now available in the lab:
+
+- [LoadingButton](/components/buttons/#loading-buttons). It does what you would expect. It renders the `Button` in a loading/pending state.
+- [TrapFocus](/components/trap-focus/). This component traps the focus within a DOM node. For instance, it's used by the Modal for accessibility reasons.
+- [Masonry](/components/masonry/). One great use case for this component is when using the `Grid` component leads to wasting space. It's frequently used in dashboards.
+
+  <a href="/components/masonry/"><img src="/static/blog/mui-core-v5/masonry.png" alt="" style="width: 505px; margin-bottom: 16px;" /></a>
```

Comment:
Should we maybe use an example with images here?

Reply:
I didn't because I felt that the [most common](https://docs.google.com/forms/d/1lOfuOc4XWcvbCH6cEcjsrZ7gvyHPzZTeV5_E9Yh-J0s/edit#responses) [use case](https://github.com/mui-org/material-ui/issues/17000) is with dashboards, and that image would be distracting. But not strong point of view.